scripts/pre_commit_linter.py

```diff
@@ -697,6 +714,12 @@ def _get_all_files():
     """
     eslintignore_path = os.path.join(os.getcwd(), '.eslintignore')
     parsed_args = _PARSER.parse_args()
+    if parsed_args.mode == 'v':
+        global MODE #pylint: disable = global-statement
```

Comment:
Please find an alternative to this. We should not be using global variables. Also, no pylint disable pragmas.

Reply:
Will change it.

Reply:
DONE .